queries/futures/usePortfolioData.ts

```diff
@@ -0,0 +1,85 @@
+import request, { gql } from 'graphql-request';
+import { useQuery, UseQueryOptions } from 'react-query';
+import { useRecoilValue } from 'recoil';
+import Wei from '@synthetixio/wei';
+
+import { isL2State, networkState, walletAddressState } from 'store/wallet';
+import { getFuturesEndpoint } from './utils';
+import EthDater from 'ethereum-block-by-date';
+import moment from 'moment';
+import { appReadyState } from 'store/app';
+import Connector from 'containers/Connector';
+
+type PortfolioData = {
+	margin: Wei;
+	block: number;
+	timestamp: number;
+}[];
+
+// The chart basically plots margin against block number.
+
+const usePortfolioData = (options?: UseQueryOptions<PortfolioData | null>) => {
```

Comment:
I'm assuming this part is still in progress?

Reply:
Yeah. Basically suspended it, since it wasn't very feasible to make users wait >20s for the monthly portfolio data to load. It's suspended till we get the IPFS stuff up and running.